addons/sys_gestures/script_component.hpp

```diff
@@ -0,0 +1,15 @@
+#define COMPONENT sys_gestures
+#define COMPONENT_BEAUTIFIED Radio Gestures
```

Comment:
```suggestion
#define COMPONENT_BEAUTIFIED Gestures
```
Radio is in the name of the mod. :D